ale/drivers/base.py

```diff
@@ -1238,7 +1300,20 @@ def _sensor_orientation(self):
         """
         if not hasattr(self, "_inst_pointing_table"):
             self._init_tables()
-        return self._inst_pointing_table.get('Rotations', None)
+
+        inst_pointing_times = np.linspace(self._inst_pointing_table["Times"][0],
+                                          self._inst_pointing_table["Times"][-1],
+                                          int(self.number_of_quaternions))
```

Comment:
Why is this cast here and not on `number_of_quaternions`? Seems like this should always be an int from that property.